---
layout: page
title: Selected projects
description: Jean Morrison's projects
---

<div class="navbar">
    <div class="navbar-inner">
        <ul class="nav">
            <li><a href="#smooth">spatially structured traits</a></li>
            <li><a href="#rcc">confidence intervals in high dimensions</a></li>
            <li><a href="#GWAS">GWAS and GWAS related methods</a></li>
        </ul>
    </div>
</div>

### <a name="smooth"></a>Assocations with spatially structured traits

Sequencing based assays used to measure epigenomic features tend to produce very dense spatially structured measurements like these bisulfite sequencing data from [ENCODE](https://www.encodeproject.org/):

![bisulfite sequencing data in a small region of chromosome 22](pics/meth_dat_example.png =100x)

We call data generated by these assays "genomic phenotypes".
Often we are interested in identifying regions of the genome where a genomic phenotype is associated with an experimental condition or an organismal level trait (like height). I've worked on two different approaches to this problem. 

In both of these projects, our motivations come from genomic data but there are lots of other spatially structured data types, like time-series or neuroimaging data, that these approaches might be applied to. 

#### Flexible Robust Excursion Test (FRET)
FRET is a method for testing associations with genomic phenotypes that can both adaptively learn the boundaries of associated regions and control the false discovery rate. 
Previous methods for solving this problem.
[![github](icons16/github-icon.png)](https://github.com/jean997/fret)



#### Joint Adaptive Differential Estimation (JADE)

JADE is a more graphical/descriptive tool for identifying associations between one spatially structured trait and one categorical trait. We use penalized likelihood to estimate smooth profiles borrowing both across positions and catgories. 
[![github](icons16/github-icon.png)](https://github.com/jean997/jadeTF)

### <a name="rcc"></a>Rank Conditional Coverage


### <a name="GWAS"></a>GWAS and GWAS related projects
